dev/CMakeLists.txt

```diff
@@ -4,26 +4,20 @@ project(Lobster)
 
 option(LOBSTER_TOCPP "Build Lobster with the output of --to-cpp" OFF)
 
-set(SDL_SHARED OFF CACHE BOOL "Build a shared version of the library")
-function(install)
-endfunction(install)
-add_subdirectory("external/SDL")
-
-set(EXE_NAME lobster)
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+  set(CMAKE_INSTALL_PREFIX ${CMAKE_SOURCE_DIR}/..)
+endif()
+set(SDL_SHARED OFF CACHE INTERNAL "")
+set(SDL_STATIC ON CACHE INTERNAL "")
+set(VIDEO_VULKAN OFF CACHE INTERNAL "")
+add_subdirectory(external/SDL EXCLUDE_FROM_ALL)
 
 file(GLOB LOBSTER_SRCS
     "src/*.h"
     "src/lobster/*.h"
     "src/*.cpp"
 )
 
-if(LOBSTER_TOCPP)
```

Comment:
Why was this removed?

Reply:
I moved it down so it's after the CMAKE_RUNTIME_OUTPUT_DIRECTORY setting, otherwise that setting won't take effect.

Reply:
TBH I didn't verify the `LOBSTER_TOCPP` case, but I assumed the `set(EXE_NAME` required `CMAKE_RUNTIME_OUTPUT_DIRECTORY`.

Reply:
Ah ok, didn't spot that.